apps/pie-docs/src/components/chip/code/code.md

```diff
@@ -76,6 +77,16 @@ We recommend using `pie-icons-webc` when using the `icon` slot. Here is an examp
 </pie-chip>
 ```
 
+## Events
+
+When the chip component is not dismissible, it does not use any custom event handlers. In order to add event listening to this component, you can treat it like a native HTML element in your application.
```

Comment:
```suggestion
When the chip component is not dismissible, it does not emit any custom events. In order to add event listening to this component, you can treat it like a native HTML element in your application.
```

Reply:
cheeky one, I copied it from other references, should I update them too? :D

Reply:
updated them too ;p